perception/object_recognition/prediction/map_based_prediction/launch/map_based_prediction.launch.xml

```diff
@@ -4,9 +4,9 @@
   <arg name="prediction_time_horizon" default="10" />
   <arg name="prediction_sampling_delta_time" default="0.5" />
   <arg name="vector_map_topic" default="/map/vector_map" />
-  <node pkg="map_based_prediction" type="map_based_prediction" name="map_based_prediction" output="screen">
-    <param name="prediction_time_horizon" value="$(arg prediction_time_horizon)" />
-    <param name="prediction_sampling_delta_time" value="$(arg prediction_sampling_delta_time)" />
-    <remap from="/vector_map" to="$(arg vector_map_topic)"/>
+  <node pkg="map_based_prediction" exec="map_based_prediction" name="map_based_prediction" output="screen">
+    <param from="$(var prediction_time_horizon)" />
+    <param from="$(var prediction_sampling_delta_time)" />
```

Comment:
These are not yaml files so `from` is not appropriate.

```suggestion
    <param name="prediction_time_horizon" value="$(var prediction_time_horizon)" />
    <param name="prediction_sampling_delta_time" value="$(var prediction_sampling_delta_time)" />
```

Reply:
fixed in the latest commit